.github/workflows/main.yml

```diff
@@ -3,7 +3,7 @@ on: [push, pull_request]
 
 jobs:
   Linux:
-    runs-on: ubuntu-24.04
+    runs-on: ubuntu-24.04-${{ matrix.architecture }}
```

Comment:
IIRC there's only `ubuntu-24.04` for x86-64 and `ubuntu-24.04-arm` for ARM64. This definitely won't work.

Reply:
now using - runner: [ubuntu-24.04, ubuntu-24.04-arm]